### DISCUSSION TITLE
Dudas Enunciado TPA3

### DISCUSSION THREAD

**Discussion by alesiosaba** [dds-jv/foro#287] [TPA - 2021] [answered]
Buenas, ​tenemos algunas dudas sobre los siguientes requerimientos: 

> _1. Se debe permitir que una persona genere una publicación para dar en adopción a su mascota._
> _Una persona puede necesitar o querer dar en adopción a su mascota por diferentes motivos._

¿Las personas que quieren dar en adopción una mascota tienen que ser usuarios registrados previamente o lo pueden hacer sin necesidad de serlo? ¿O cualquier persona que utilice la página puede publicar una mascota para que luego sea adoptada?

En caso de que tengan que ser usuarios registrados ¿Las mascotas que serán dadas en adopción son las mascotas que fueron registradas en el sistema anteriormente en caso de que se pierdan?

> _2. Se debe permitir que cada organización defina las preguntas que le realizará al dueño cuando éste quiera dar en adopción a su mascota. Se debe tener en cuenta que las preguntas podrían llegar a variar en cualquier momento._

¿Las preguntas sobre las necesidades de las mascotas y comodidades que ofrece el adoptante pueden ser solo de tipo bool? Como por ejemplo si cuenta o no con patio, si cuenta o no con otros animales, etc. ¿Debemos considerar también otros tipos de necesidades como "cantidad máxima de mascotas con las que puede convivir" siendo en ese caso de tipo int? 

Por otro lado ¿Las preferencias de los adoptantes hacen referencia a las características ya existentes de las mascotas como pueden ser el tipo de animal (perro o gato), edad del animal, color?

> 4. Se debe permitir que una persona genere una publicación que demuestre su intención de adoptar una mascota, teniendo en cuenta sus preferencias y comodidades.

En el caso en que una persona tenga intención de adoptar una mascota ¿Esta persona debe ser un usuario registrado previamente en el sistema o cualquier persona que utilice la página se puede postular como adoptante?

**Comment by gastonprieto** [comment, marked as answer]
Lo cierro porque me imagino que esto ya quedo hablado con el ayudante
